docs/installation/frameworks/vuejs-v3.md

```diff
@@ -235,15 +235,59 @@ Finally, import the build in your application:
 
 ## Using CKEditor from source
 
-Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}.
+Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}. You have two options regarding building your application: Vite or Webpack.
+
+### Vite
+
+This guide assumes that you use `create-vue` as your boilerplate. To get started with Vite and Vue, run the command below.
+
+```bash
+npm init vue@latest ckeditor5-vue-example
+```
+
+This command will install and execute `create-vue`, the official project scaffolding tool for Vue. It will also allow you to customize your project, for example, by adding Typescript. Choose your preferred options.
+
+#### Installing necessary packages
+
+You need two packages to use CKEditor from source with Vue and Vite: the official Vue component and the Vite plugin.
+
+<info-box>
+	Using the Vite plugin to build CKEditor 5 from the source in Vite is still in the experimental phase. We encourage you to test it and give us feedback. To read more about integration with Vite or its limitations, check the {@link installation/advanced/integrating-from-source-vite Integrating from source with Vite} guide.
+</info-box>
+
+Install necessary packages alongside the default theme using the following command.
+
+```bash
+npm install --save @ckeditor/vite-plugin-ckeditor5 @ckeditor/ckeditor5-vue @ckeditor/ckeditor5-theme-lark
+```
+
+#### Configuring `vite.config.js`
+
+Configuring CKEditor with Vue and Vite is simple. Modify the existing config by importing `ckeditor5` and adding it to the list of plugins.
+
+```js
+// vite.config.js
+
+import ckeditor5 from '@ckeditor/vite-plugin-ckeditor5';
+
+export default defineConfig( {
+	plugins: [
+		ckeditor5( { theme: require.resolve( '@ckeditor/ckeditor5-theme-lark' ) } )
```

Comment:
In Vue3 apps that are using Vite, there will be vue() plugin used as well. We can show vite.config.js example with that plugin, as this will be only 2 lines more and it will show that ckeditor5() is added along with other vite plugins in this config.